core/chains/evm/config/toml/config.go

```diff
@@ -412,8 +412,16 @@ func (c *Chain) ValidateConfig() (err error) {
 					err = multierr.Append(err, commonconfig.ErrInvalid{Name: "Transactions.AutoPurge.DetectionApiUrl", Value: c.Transactions.AutoPurge.DetectionApiUrl.Scheme, Msg: "must be http or https"})
 				}
 			}
-		case chaintype.ChainZkEvm:
-			// No other configs are needed
+		case chaintype.ChainZkEvm, chaintype.ChainXLayer:
+			// MinAttempts is an optional config that can be used to delay the stuck tx detection for zkEVM or XLayer
+			// If MinAttempts is set, BumpThreshold cannot be 0
```

Comment:
what's the reason for `If MinAttempts is set, BumpThreshold cannot be 0` ?

Reply:
`BumpThreshold=0` disables gas bumping so transactions would never have bump attempts. So if gas bumping was disabled, we'd never meet the `MinAttempts` criteria